lms/static/scripts/frontend_apps/components/dashboard/AssignmentActivity.tsx

```diff
@@ -100,9 +100,13 @@ export default function AssignmentActivity() {
             case 'replies':
               return <div className="text-right">{stats[field]}</div>;
             case 'last_activity':
-              return stats.last_activity
-                ? formatDateTime(new Date(stats.last_activity))
-                : '';
+              return stats.last_activity ? (
+                <div className="whitespace-nowrap">
+                  {formatDateTime(new Date(stats.last_activity))}
+                </div>
```

Comment:
Perhaps worth extracting this block into a component, since it is repeated in several places. That can also cache the string -> date -> string conversion.